bps/templates/bps_darkBootstrap.yaml

```diff
@@ -2,6 +2,13 @@ pipelineYaml: "${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpDarkBootstrap.yaml"
 
 project: "${TICKET}"
 campaign: "${TICKET}"
+submitPath: "$SCRATCH/submit/{outputRun}"
```

Comment:
If we are standardizing on braces then it should be `${SCRATCH}` here and elsewhere